.circleci/config.yml

```diff
@@ -64,15 +64,15 @@ commands:
       - run:
           name: "Decode creds"
           command: |
-            echo "export const token ='$(curl $ACCESS_TOKEN_URL)'" >> test-application/src/e2e-tests-token.ts
+            echo "export const token ='$(curl $ACCESS_TOKEN_URL)'" >> test/app/src/e2e-tests-token.ts
   clear-detox-cache:
     description: "Clears detox framework cache"
     steps:
       - run:
           working_directory: .
           name: Clear detox cache
           command: |
-            cd test-application
+            cd test/app
             detox clean-framework-cache
             detox build-framework-cache
             cd ../
```

Comment:
For all the `cd ../` commands here and below, do we need to go one more level up?